StripePaymentSheet/StripePaymentSheet/Source/Analytics/PaymentSheetAnalyticsHelper.swift

```diff
@@ -133,6 +133,13 @@ final class PaymentSheetAnalyticsHelper {
             let linkMode: String = elementsSession.linkPassthroughModeEnabled ? "passthrough" : "payment_method_mode"
             params["link_mode"] = linkMode
         }
+        if elementsSession.customer?.customerSession != nil {
```

Comment:
@wooj-stripe how do we handle customer session analytics today? Why don't we just log the whole thing or have it define an "analyticPayload", similar to PaymentSheet.Configuration.

Reply:
Currently we only send `customer_access_provider` which describes `ek` vs `cs` - but agree -- we can send the deserialized values of each feature as well!